app/crud/microserviceTeamScoreCard.py

```diff
@@ -0,0 +1,52 @@
+import sqlalchemy
+from sqlalchemy.orm import Session
+from ..models import Microservice
+from ..schemas import MicroserviceTeamScorecardBase,MicroserviceTeamScorecardCreate,MicroserviceTeamScorecardUpdate, team, scoreCard
+from .base import CRUDBase
+from typing import List
+from sqlalchemy.sql import func
+from starlette.exceptions import HTTPException
+from . import CRUDTeam, CRUDMicroserviceScoreCard, CRUDScoreCard
+
+
+
+class CRUDMicroserviceTeamScorecard(CRUDBase[MicroserviceTeamScorecardBase, MicroserviceTeamScorecardCreate, MicroserviceTeamScorecardUpdate ]):
```

Comment:
This is not a class that directly operate on a single DB table. You shouldn't make it inherit from the CRUDBase class.
ONLY classes that directly operate on a SINGLE  Database table can inherit form CRUDBase class.

This is a utility class more than a CRUD class, And I believe there's no need to make it a separate class. You only use  `getByServiceIdWithTeamAndSCOREDetails` method. Why don't you create a new file inside a folder called utils or helpers for example and move this `getByServiceIdWithTeamAndSCOREDetails` function in that file.